src/components/breadcrumb/breadcrumb.tsx

```diff
@@ -4,31 +4,32 @@ import { Icon } from '../icon'
 import { elMr2 } from '../../styles/spacing'
 import { FlexContainer } from '../layout'
 
-export interface BreadCrumbItem extends HTMLAttributes<HTMLDivElement> {
+export interface BreadCrumbItem extends HTMLAttributes<HTMLAnchorElement> {
   text: string
   onClick: () => void
 }
 
-export interface BreadCrumbProps extends HTMLAttributes<HTMLDivElement> {
+export interface BreadCrumbProps extends HTMLAttributes<HTMLElement> {
   items: BreadCrumbItem[]
   defaultActiveIndex?: number
 }
 
-export const handleNext = (setActive: Dispatch<SetStateAction<number>>, onClick: () => void, index: number) => (
-  e: MouseEvent<HTMLDivElement>,
-) => {
-  e.preventDefault()
-  setActive(index)
-  onClick()
-}
+export const handleNext =
+  (setActive: Dispatch<SetStateAction<number>>, onClick: () => void, index: number) =>
+  (e: MouseEvent<HTMLAnchorElement>) => {
+    e.preventDefault()
+    setActive(index)
+    onClick()
+  }
 
 export const BreadCrumb: FC<BreadCrumbProps> = ({ items, defaultActiveIndex = 0, ...rest }) => {
   const [active, setActive] = useState<number>(defaultActiveIndex)
 
   return (
     <ElBreadCrumbContainer {...rest} aria-label="Breadcrumb">
       {items.map(({ onClick, text }, index) => {
-        if (index > active) return null
+        // Why is this line here and why is it causing a problem now?
+        // if (index > active) return null
```

Comment:
Not sure why you added this condition? https://github.com/reapit/foundations/blame/master/packages/elements/src/components/breadcrumb/breadcrumb.tsx#L31

Reply:
Because if the items are 

Item 1 | item 2 | item 3

And my position on a page is item 2 and therefore the active value is 2, I don't want to render item 3 because that's not where my current navigation position is.

Reply:
It's probably the story that needs fixing ie set a default active value